hostutils/psu.md

```diff
@@ -1,9 +1,117 @@
-# Phoenix Software Uploader (psu)
+# Phoenix Serial Uploader (psu)
 
-Documentation under development.
+`psu` is a tool designed to load firmware onto Phoenix-RTOS devices using the SDP (Script Download Protocol) protocol.
```

Comment:
I see that it's the same content as in the `psu` README, so please place the link over there like it's done in https://github.com/phoenix-rtos/phoenix-rtos-ports/tree/46f7c3d1449f0aa8d9b72457a53b2c87faa047d6/azure_sdk

Applies to the other places, where you just copied readme.